requirements.txt

```diff
@@ -3,4 +3,5 @@ ops>=1.3.0,<2.0.0
 lightkube>=0.10.1,<1.0.0
 jsonschema
 pyyaml
-git+https://github.com/canonical/ops-lib-manifest.git@main
\ No newline at end of file
+git+https://github.com/canonical/ops-lib-manifest.git@main
+git+https://github.com/charmed-kubernetes/interface-kube-control.git@ops-package/requires#subdirectory=ops
```

Comment:
THIS CANNOT MERGE

Reply:
Need this merged first
* https://github.com/juju-solutions/interface-kube-control/pull/36